crypto/params.c

```diff
@@ -47,8 +47,8 @@ int OSSL_PARAM_modified(const OSSL_PARAM *p)
 
 void OSSL_PARAM_set_unmodified(OSSL_PARAM *p)
 {
-    if (p != NULL)
-        p->return_size = OSSL_PARAM_UNMODIFIED;
+    while (p != NULL)
```

Comment:
It should be p->key != NULL

Reply:
or `p != NULL && p->key != NULL`

Reply:
Lesson to self: don't try to rework code when very tired.
It's never stuck before, I doubt it will this time either :(

Thanks.

Reply:
Fixed now.